opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -412,9 +412,8 @@ def get_task_value():
 def namespaced_view_name(view_name, metric_prefix):
     """ create string to be used as metric type
     """
-    if metric_prefix == "":
-        return os.path.join("custom.googleapis.com", "opencensus", view_name)
-    return os.path.join(metric_prefix, view_name)
+    metric_prefix = metric_prefix or "custom.googleapis.com/opencensus"
+    return os.path.join(metric_prefix, view_name).replace('\\', '/')
```

Comment:
using os.path.join here may not be ideal. Perhaps use `join` with the specified separator instead?

Reply:
os.path.join will take care of duplicated/trailing separators, for example, join("a/b/", "c") will yield the same result as join("a/b", "c"). It'll be better to reuse such logic from os.path.join than manually doing path normalization and joining.